CHANGELOG.md

```diff
@@ -182,6 +198,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - The deprecated `@cumulus/common/string.replace` functon has been removed
 - The deprecated `@cumulus/common/string.toLower` functon has been removed
 - The deprecated `@cumulus/common/string.toUpper` functon has been removed
+- The deprecated `@cumulus/common/util.setErrorStack` function has been removed
```

Comment:
Why isn't `uuid` also listed here?

Reply:
I hadn't listed it here since it was already listed under "Breaking Changes", but it probably should be. I'll fix that.